### PR TITLE
feat: Add  a simple Calculaor.

### PR DESCRIPTION
**Pull request by adithyanmadhi**
# Add  a simple Calculaor.
Screenshot 2024-02-02 025623

Screenshot 2024-02-02 025651

# Jira link
https://mahendrankannan.atlassian.net/browse/ON-29

### CODE REVIEW

File: web-application/simple-calculator/desktop.ini
Comment (gkmahendran09) [inline web-application/simple-calculator/desktop.ini:1]: We can remove this file.

File: web-application/simple-calculator/script.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    console.log(val)
+    document.getElementById("screen").value=document.getElementById("screen").value+=val;
+}
+
+function cleardisplay(){
+    document.getElementById("screen").value=""
+}
+function equalclick(){
+    var text=document.getElementById("screen").value
+    var result=eval(text)
```
Comment (gkmahendran09) [inline web-application/simple-calculator/script.js:12]: Avoid using `eval` in unsecured way.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!

**Review by gkmahendran09 — APPROVED** [review]

**Review by gkmahendran09 — CHANGES_REQUESTED** [review]
Please fix the comments

File: web-application/simple-calculator/desktop.ini
Comment (adithyanmadhi) [reply web-application/simple-calculator/desktop.ini:1]: Removed

File: web-application/simple-calculator/script.js (hunk 1 above)
Comment (adithyanmadhi) [reply web-application/simple-calculator/script.js]: Removed eval()

**Review by gkmahendran09 — APPROVED** [review]

**Pull request merged**